uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml

```diff
@@ -17,11 +17,10 @@ ComponentConfiguration:
     controlPort: 47619
 Manifests:
   - Artifacts:
-      - URI: classpath:/local-store/artifacts/client-python-paho
+      - URI: classpath:/local-store/artifacts/client-python-paho.exe
```

Comment:
can we have two manifests, one for linux, one for windows?

Reply:
At first, we created two manifests, but then we discovered that the OTF has a bug related to it.
It cannot parse platform-depending manifests at this moment.
Therefore, we decided to make the same name for both platforms to make it possible to run tests with this client on them.
When the bug will be fixed, we will update this recipe to have 2 manifests.

Reply:
Please fix the bugs

Reply:
Ok, this PR will wait for fix

Reply:
Maybe has no sense to wait for multi-platform support in OTF in that case due to we should customize artifact copying in maven. So will do some tricks in any case